doc/source/parameters/cfd/linear_solver_control.rst

```diff
@@ -119,10 +119,12 @@ In this subsection, the control options of linear solvers are specified. These c
 * The ``minimum residual`` for the linear solver.
 
 * The ``relative residual`` for the linear solver.
+
```

Comment:
that's strange I did not have any problem with the formatting on my part, thank you for spotting and correcting it!

Reply:
It's no problem! I also had problems that others didn't encounter, I don't really know why...